### ISSUE TITLE
Define data entities

### ISSUE THREAD

**Issue by zverev** [Crisismap/crisismap-backend#5]
![entities](https://cloud.githubusercontent.com/assets/5755474/21157901/31d48160-c18c-11e6-9a0e-4eed6386f50b.PNG)

According to our excel entity table, I don't understand the meaning of `news` entity. Is it a media data from sources?
Why not always use `incident` to store an object, that describes some kind of event?
By the way I totally don't like the word `news`. It isn't clear, how to call a single object. `news` is always plural.

Here are possible data types for client ([flowtype](https://flowtype.org/)):

```js
type Location = {
  id: Number,
  geometry: GeoJSON,
  population: Number
}

type Toponym = {
  id: Number,
  name: String,
  locationId: Number
}

type IncidentType = 'fire'|'flood'|'earthquake'

// what we get from workers
type Media = {
  title: String,
  description: String,
  sourceUrl: String
  // something else
}

type Incident = {
  id: Number,
  locations: Array<Location>,
  type: IncidentType,
  registeredAt: Date,
  media: Media
}
```

Can `Incident` have multiple `type`s?

**Comment by Godnik** [comment]
I agree about ```news``` unconvincing definition - it's better to define this type of data as ```media```, minding that the social media sources can be also represented as data suppliers.
The ```incident``` is produced as a derivative layer on the top of this data. So I'd propose smth like this with an eye to the future:
```js
type Incident = {
  id: Number,
  locations: Array<Location>,
  type: IncidentType,
  registeredAt: DateTime,
  media: Media,
  weatherAlerts: weatherAlerts,
  fireAlerts: fireAlerts,
  eqAlerts: eqAlerts,
  ecoAlerts: ecoAlerts,
  significant: Number,
  populationSum: Number,
  status: String
}
```

**Comment by bkmy43** [comment]
Guys I agree, that _news_ is not perfect term, because it has no singular form. But proposed _media_ is in my humble opinion very misleading.

According to our definition _news_ are items, which we get from our sources, they are messages with particular information which we get from different _media_. And "media" itself is just a channel like "newpaper x" or "website y", not the information we get.

So I would love to use any other term for _news_ if we find something which is clear and has both singular and plural forms
